src/test/resources/History.feature

```diff
@@ -0,0 +1,31 @@
+Feature: History
+
+  @automated
+  Scenario: No history
+    Given user is in Katalon login page
+    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
+    And user clicks on login button
+    Then user sees appointment page
```

Comment:
cái này c có thể viết gọn lại, vì mh phải sử dụng ở rất nhiều scenario về sau, nếu cứ để 4 dòng thì scenario nó sẽ rất dài
   ` Givenuser signed in with username as "John Doe" and password as "ThisIsNotAPassword"`
còn check nó vào trang web sau khi login thành công thì đến đâu đã được check ở login rồi, không cần phải check lại ở đây